docs/managing-dependencies.md

```diff
@@ -36,6 +36,7 @@ on whether their dependencies will be resolved and installed **by default**, the
 the dependencies logically.
 {{% /note %}}
 
+{{% note %}}
 The dependencies declared in `tool.poetry.dependencies` are part of an implicit `default` group.
```

Comment:
```suggestion
The dependencies declared in `tool.poetry.dependencies` are part of an implicit `default` group.
In other words, `tool.poetry.dependencies` is equivalent to `tool.poetry.group.default.dependencies`.
```

In the other issue (in poetry-core) we have come to the conclusion that it is sensible to mention the explicit default group.

Reply:
Oh yes, I didn't consider that. As long as it can be misused, it's probably better not to mention it explicitly.